### DISCUSSION TITLE
Implement Updatescript as bash alias

### DISCUSSION THREAD

**Discussion by KaiserN** [tteck/Proxmox#778] [Ideas]
Hi,

would be nice if you can implement like unifi update script as an bash alias like "unifi-update" or so

greetings

**Comment by tteck** [comment]
What???

**Reply by KaiserN** [reply]
like this

.bashrc
`
alias unifi-update='bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/unifi-update.sh)"'`
